bin/deploy

```diff
@@ -59,15 +55,15 @@ createDirectories
 if isInstalled 'collectiveaccess/providence'; then
     echo "Deploying Providence"
     ln -sf vendor/collectiveaccess/providence $COLLECTIVEACCESS_HOME
-    composerInstall $COLLECTIVEACCESS_HOME
+    linkDirectories $COLLECTIVEACCESS_HOME
     deployProvidence
 fi
 if isInstalled 'collectiveaccess/pawtucket2'; then
     echo "Deploying Pawtucket"
     PAWTUCKET_SOURCE_PATH=$APP_ROOT/pawtucket-theme
     pushd $PAWTUCKET_SOURCE_PATH
     ln -sf $PAWTUCKET_SOURCE_PATH/vendor/collectiveaccess/pawtucket2 $PAWTUCKET_HOME
-    composerInstall $PAWTUCKET_HOME
+    linkDirectories $PAWTUCKET_HOME
```

Comment:
PROV doesn't have pawtucket, so I can't test this, but logically it should work in the same way?